src/components/ActivityListItem.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import activities from '../activities.json'
+import styled from 'styled-components'
+import ShowMoreText from 'react-show-more-text'
+
+export default function ActivityList() {
+  return (
+    <>
+      {activities.map(activity => {
+        return (
+          <li key={activity.name}>
+            <DivisionLine data-testid="division-line" />
```

Comment:
Damit du in der Attribut-Schreibweise konsistent bleibst, würde ich data-test-id schreiben

Reply:
Gerade angepasst und dann einen Fehler bekommen, dass nichts über data-test-id gefunden wird. Anscheinend ist data-testid die korrekte/notwendige Schreibweise.